### DISCUSSION TITLE
Encryption password

### DISCUSSION THREAD

**Discussion by erottensteiner** [remotely-save/remotely-save#641] [General]
Why is the encryption password for E2E-encryption (e.g. for onedrive) stored in plain text locally?
I think this could be a great security problem? Why is this - does anyone know?

**Comment by fyears** [comment]
Because the plugin assumes your local device is secure and you trust Obsidian and all your plugins. In this case, no problem at all.

On the other hand, if you don't trust your local device, you should run some virus scanning software. If you don't trust Obsidian or any plugins, you should uninstall them.

**Comment by erottensteiner** [comment]
Yes, that's true - I only thought on a situation if you loose your device (or your device was stolen) - then it would be better that for access to the vault the password must be entered (or verification by faceid/fingerprint). I know, that's not an important thing, but maybe a good idea for the future...?

Btw. - I've forgotten: remotely-save is a wonderful peace of software and works perfect for me - so my wish should only be an additional idea...

**Reply by fyears** [reply]
> then it would be better that for access to the vault the password must be entered (or verification by faceid/fingerprint)

manually entering the password is indeed more secure but i am afraid that makes users not wanting to use the plugin due to the inconvenience, especially when some users enable the auto sync every minute...

**Reply by erottensteiner** [reply]
Hi fyears,
thank you for your answer - thinking about obsidian and the storage of files, I was on a wrong way. Problem is not synching and remotely-save - someone must take care that the local MD-files are protected (e.g. encryption of local hard drive). From this direction everything is clear and sorry for the inconvenience, remotely-save does its job exactly, my requirement can be solved in other way...
Best regards
(discussion can be closed... ;-))
